Backend/DbSeed/DbSeed.cs

```diff
@@ -220,15 +220,16 @@ public static async Task LoadTestingCoursesAsync(IServiceProvider serviceProvide
                 {
                     Id = Guid.NewGuid(),
                     CourseCode = crs.CourseCode,
+                    CourseShortcut = crs.CourseShortcut,
                     CourseName = crs.CourseName,
                     LastUpdateOfTimetable = null,
                     YearOfStudy = crs.YearOfStudy,
-                    StudyType = crs.StudyType
+                    StudyType = crs.StudyType.Trim()
                 };
-                Course c = await courseCollection.Find(x => x.CourseCode == crs.CourseCode && x.YearOfStudy == crs.YearOfStudy && x.StudyType == crs.StudyType).SingleOrDefaultAsync();
-                if (c != null) continue;                
-                await courseCollection.InsertOneAsync(course);                
-            }            
+                Course c = await courseCollection.Find(x => x.CourseShortcut == null && x.CourseCode == crs.CourseCode && x.YearOfStudy == crs.YearOfStudy && x.StudyType == crs.StudyType.Trim()).FirstOrDefaultAsync();
```

Comment:
Are you sure `x.CourseShortcut == null`?

Reply:
Thank you, i missed this one, im going to fix it.